src/components/application_manager/src/resumption/resume_ctrl.cc

```diff
@@ -381,11 +383,15 @@ bool ResumeCtrl::CheckPersistenceFilesForResumption(
   bool result = resumption_storage_->GetSavedApplication(
       application->policy_app_id(), device_mac, saved_app);
   if (result) {
-    if (!CheckIcons(application, saved_app[strings::application_commands])) {
-      return false;
+    if (saved_app.keyExists(strings::application_commands)) {
+      if (!CheckIcons(application, saved_app[strings::application_commands])) {
+        return false;
+      }
     }
-    if (!CheckIcons(application, saved_app[strings::application_choice_sets])) {
-      return false;
+    if (saved_app.keyExists(strings::application_choice_sets)) {
+      if (!CheckIcons(application, saved_app[strings::application_choice_sets])) {
+        return false;
```

Comment:
@VVeremjova maybe it's better to do in such way **if( statement1 && !statement2)** istead of **if(statement1) { if(statement2)...**